collections/_article/parallax-progress-report.md

```diff
@@ -0,0 +1,80 @@
+
+- Ability to follow camera rotation
+- Compatibility with `CanvasGroup` and `BackBufferCopy`
+- Support for zooming out with a `Camera2D`
+- A simpler workflow
+- Autoscroll
+- Performance improvements
+- Better documentation
+
+#### Reogranization
+
+![The Parallax2D inspector](/storage/blog/parallax2d/inspector.webp)
+
+The inspector properties are now reorganized and categorized. Notice that the limits now follow the same logic as `Camera2D` limits. There are notes in the previous parallax system's documentation clarifying that `mirroring` doesn't actually mirror anything, but rather just repeats. The name has been changed to `repeat_size` to better reflect what it actually does.
+
+#### Consolidation
+
+Requiring two nodes in a strict hierarchy has been a point of confusion and frustration for users in the past, with it not always being the clearest which node is responsible for what. With the `CanvasLayer` trick no longer required for the "infinite repeat" effect, it frees us up for some simplification. We discussed this decision in depth, and have a few reasons for it.
+
+If the new `Parallax2D` were required to be a direct child of something like `ParallaxBackground`, the only purpose it'd serve is to run a for-loop over all its children for setting values. A nice shortcut, but it isn't enough to warrant a whole separate core node and enforce that relationship. Some new handy editor shortcuts were provided in the years since the parallax system was first put in place, like the ability to select multiple of the same node and update shared properties all at once. If a user wants to update all the nodes via code, they'd need to write a script for it anyway, so it's best left up to the user on a case-by-case basis.
+
+#### `CanvasLayer` to `Node2D`
+
+You can still get the benefits of a `CanvasLayer`. The difference now is that you can choose. Just like any normal `Node2D`, placing a `Parallax2D` in a `CanvasLayer` will prevent zoom and rotating with the camera. You'll just want to make sure the `Parallax2D`'s `follow_viewport` value matches the `CanvasLayer`'s `follow_viewport_enabled` property. At this point you may notice that `Parallax2D`'s `follow_viewport` is enabled by default. This was a decision made because `ParallaxBackground`'s `follow_viewport_enabled` is not enabled even though it is following the viewport. It's confusing for a `CanvasLayer` to not behave like a `CanvasLayer`, so `Parallax2D` behaves more like one would expect.
+
+#### `autoscroll`
+
+`autoscroll` scrolls the layer automatically in addition to the parallax settings. This could be done with a custom script, but decided to do so anyway since it was highly requested, easy to implement, and had no performance hit if you weren't using it.
```

Comment:
```suggestion
`autoscroll` scrolls the layer automatically in addition to the parallax settings. This could be done with a custom script, but we decided to put this into `Parallax2D` since it was highly requested, easy to implement, and has no performance hit if you aren't using it.
```